build.sbt

```diff
@@ -3,29 +3,27 @@ name := "jira-exception-processor"
 scalaVersion := "2.11.6"
 
 resolvers ++= Seq(
-  "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/",
-  rhinoflyRepo("RELEASE").get
+  "Typesafe repository" at "http://dl.bintray.com/typesafe/maven-releases/",
```

Comment:
Waarom deze? Als het goed is staan alle libraries van typesafe gewoon op de `maven.org` repository dus is deze overbodig.